sys/contrib/dev/iwlwifi/iwl-trans.h

```diff
@@ -252,7 +252,7 @@ struct iwl_rx_cmd_buffer {
 
 static inline void *rxb_addr(struct iwl_rx_cmd_buffer *r)
 {
-	return (void *)((unsigned long)page_address(r->_page) + r->_offset);
+	return (void *)((uintptr_t)page_address(r->_page) + r->_offset);
```

Comment:
Maybe use 'iwlwifi rxb_addr:' as the tag in the commit log since this is specific to that driver.

This is also a change you could send to @baz21 to possibly get merged upstream.

Reply:
Reworded!